### PR TITLE
PPR UI new registration step 1, fee summary, stepper. Also 7778 UI non-prod banner.

### PR DESCRIPTION
**Pull request by doug-lovett**
*Issue #:* /bcgov/entity/#7026

*PR UI new registration step 1, fee summary, stepper. Also 7778 UI non-prod banner.*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by doug-lovett** [comment]
I should have added that some tests are pending.

File: ppr-ui/src/components/common/RegistrationFee.vue [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { useGetters } from 'vuex-composition-helpers'
+
+import { FeeSummaryIF } from '@/interfaces' // eslint-disable-line no-unused-vars
```
Comment (jordiwes) [inline ppr-ui/src/components/common/RegistrationFee.vue:51]: There is a Fee  Summary shared component... worth using that?

File: ppr-ui/src/components/registration/RegistrationLengthTrust.vue [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    const feeSummary: FeeSummaryIF = getFeeSummary.value
+    const localState = reactive({
+      registrationType: props.defaultRegistrationType,
+      trustIndenture: lengthTrust.trustIndenture,
+      lifeYearsDisabled: false,
+      lifeInfinite: lengthTrust.lifeInfinite,
+      lifeYearsEdit: (lengthTrust.lifeYears > 0) ? lengthTrust.lifeYears.toString() : '',
+      lifeYearsMessage: '',
+      trustIndentureHint: '',
+      lifeYearsHint: 'Minimum 1 year, Maximum 25 years ($5.00 per year)',
```
Comment (jordiwes) [inline ppr-ui/src/components/registration/RegistrationLengthTrust.vue:92]: 5.00, always wonder if that should be a constant somewhere for easy refactor :)

File: ppr-ui/src/components/registration/RegistrationLengthTrust.vue [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      setLengthTrust(lengthTrust)
+    })
+    watch(() => localState.lifeInfinite, (val: boolean) => {
+      // localState.lifeYearsDisabled = localState.lifeInfinite
+      lengthTrust.lifeInfinite = val
+      if (lengthTrust.lifeInfinite) {
+        localState.lifeYearsEdit = ''
+        lengthTrust.lifeYears = 0
+        feeSummary.quantity = 1
+        feeSummary.feeAmount = 500.00
```
Comment (jordiwes) [inline ppr-ui/src/components/registration/RegistrationLengthTrust.vue:142]: same with 500 (constant somewhere)

File: ppr-ui/src/components/common/RegistrationFee.vue (hunk 1 above)
Comment (doug-lovett) [reply ppr-ui/src/components/common/RegistrationFee.vue:51]: I did look at that and at name requests but it does not quite work with the ppr UI wireframe.

File: ppr-ui/src/components/registration/RegistrationLengthTrust.vue (hunk 2 above)
Comment (doug-lovett) [reply ppr-ui/src/components/registration/RegistrationLengthTrust.vue:92]: Agreed, as per our discussion this morning, the fees will be refactored as a configurable property.

File: ppr-ui/src/components/registration/RegistrationLengthTrust.vue (hunk 3 above)
Comment (doug-lovett) [reply ppr-ui/src/components/registration/RegistrationLengthTrust.vue:142]: Same comment as above.

**Pull request merged**